src/compas/datastructures/network/operations/join.py

```diff
@@ -42,10 +45,88 @@ def network_join_edges(network, key):
     # set attributes based on average of two joining edges?
     network.add_edge(a, b)
 
+def network_polylines(network, splits = []):
```

Comment:
no space around `=` in assignment of default values of function arguments

Reply:
also, using a mutable object as default value can have side effects. better to use `None`